webapp/package.json

```diff
@@ -22,6 +22,9 @@
 		"cypress:open": "cypress open"
 	},
 	"dependencies": {
+		"@fortawesome/fontawesome-svg-core": "^1.2.35",
```

Comment:
I don't know if we want to add fontawesome as a dependency. Actually, we should be reusing the mattermost icons font.

Reply:
I've replaced FontAwesome with Compass Icons now.